engines/bastion_katello/app/assets/javascripts/bastion_katello/host-collections/details/views/host-collection-add-hosts.html

```diff
@@ -44,7 +44,7 @@
           </a>
         </td>
         <td bst-table-cell >{{ host.content_facet_attributes.lifecycle_environment.name}}</td>
-        <td bst-table-cell >{{ host.content_facet_attributes.name}}</td>
+        <td bst-table-cell >{{ host.content_facet_attributes.content_view.name}}</td>
```

Comment:
This seems more correct than `content_view_name`. What makes me say that is we added 'content_view' directly on the API response after `content_view_name`, so I'd interpret that to mean `content_view_name` is the legacy attribute